galaxy_skeleton2.R

```diff
@@ -79,10 +100,12 @@ variable_names <-
 # super for this kind of search-and-replace. Replace the question mark below.
 
 comma_separated_values <- 
-  ? %>% 
+  raw_file[L+1:length(raw_file)] %>% 
```

Comment:
L+1 should be in parantheses (). Not including this, results in NAs in your final data frame  "galaxies"